cmd/juju/model/destroy.go

```diff
@@ -101,18 +101,18 @@ const destroyExamples = `
 `
 
 var destroyModelMsg = `
-This command will destroy the %q model and all its resources. It cannot be stopped.`[1:]
+This command will destroy the %q model and affect the following resources. It cannot be stopped.`[1:]
 
 var destroyModelMsgDetails = `
 {{- if gt .MachineCount 0}}
  - {{.MachineCount}} {{if .IsCaaS}}container{{else}}machine{{end}}{{if gt .MachineCount 1}}s{{end}} will be destroyed
```

Comment:
The "IsCaaS" attribute is a mistake here. I am not sure why it was added, since k8s models always have machine count = 0. 
Can you please remove it and the line in `printDestroyWarningDetails` where it is passed in.

Reply:
Done